handbook/engineering/batch-changes/ce-onboarding.md

```diff
@@ -0,0 +1,62 @@
+- be comfortable demoing Batch Changes to developers and answering questions
+
+## Prerequisites
+General [CE onboarding](../../ce/onboarding.md)
+
+## Part 1: Discover Batch Changes
+- Watch the [demo video](https://www.youtube.com/watch?v=eOmiyXIWTCw) and checkout the [landing page](https://about.sourcegraph.com/batch-changes/)
+- What is Batch Changes, and why does it matter to customers? Read about [product positioning and messaging](../../marketing/batch_changes_positioning.md)
+- How to demo Batch Changes? Watch the [Batch Changes deep dive](https://docs.google.com/presentation/d/1CN3KQf1Hfdb4RO6FgBgKuiHK4ERcOAHPgVnOcBu-MPU/edit#slide=id.ga366db8d9b_0_116)
+- How does Batch Changes work? Read the Batch Changes documentation section of the [Supporting Batch Changes guide](./supporting-batch-changes.md#batch-changes-documentation)
+
+
+## Part 2: Use Batch Changes
+
+- If you are blocked at any time during onboarding, ask for help in [#batch-changes](https://sourcegraph.slack.com/archives/CMMTWQQ49).
+- Feel free to document your onboarding experience and share it back with the Bacth Changes team so that we can make this even better!
+- In this tutorial, we will use [k8s.sgdev.org](https://k8s.sgdev.org) as our environment. You should have access by default using SSO.
+- Tip: Use the docs extensively to complete the tutorial steps.
+
+### Step 1: Quickstart
+Let's start with a quick setup example to warm up: [Quickstart](https://docs.sourcegraph.com/batch_changes/quickstart).
+
+Answer those questions:
+- what does `on` do?
+- what are `steps`?
+- what is the difference between *apply* and *publish*?
+
+### Step 2: comby
+Let's use [comby](https://comby.dev/). Batch Changes can run any code change tool, but comby is great for structural-type change. See how it works with Batch Changes in this [tutorial](https://docs.sourcegraph.com/batch_changes/tutorials/refactor_go_comby). Don't publish your changesets yet!
+
+Answer those question:
+- what is comby?
+- does Batch Changes rely on comby?
+
+
+### Step 3: The `on` attribute
+Can you scope down your batch change to only a few repos? Modify the spec to scope down your batch change to <repo1> <repo2> <repo3>. Tip: you may find [`on`](https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#on) useful.
+
+### Step 4: Publishing
+Sometimes you want to publish only _some_ but not all changesets to the codehosts and keep the others unpublished, for example to test out if the repository owners will merge the changesets. Find out how to publish only on <repo> using the [`published`](https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#changesettemplate-published) attribute
+
+### Step 5: Templating
+Can you use [templating](https://docs.sourcegraph.com/batch_changes/references/batch_spec_templating) to TODO:find an approachable use case compatible with the comby example.
```

Comment:
Why are we emphasizing Comby here? I know the team likes it, but our users don't actually deeply engage with it, generally. They ask more frequently about library upgrades, built-in linters, etc.

Reply:
(I don't feel strongly about this, and there may be another reason we're emphasizing it, but just in terms of what customers often ask about.)

Reply:
Would just running a go linter offer more value?

Reply:
I think potentially! People respond well to the Dockerlint campaign we have, and often ask about ESLint, at least in my experience. It's possibly worth checking this assumption with the rest of CE, though—depending on the customer we see, we sometimes have really variable experiences of what people want.

Reply:
I think running a linter over the files returned by a search result would still allow the user to make use of templating and be useful, see first example here: https://docs.sourcegraph.com/batch_changes/references/batch_spec_cheat_sheet#loop-over-search-result-paths-in-shell-script